eahub/profiles/index.py

```diff
@@ -1,39 +1,43 @@
-from django.conf import settings
 from algoliasearch_django import AlgoliaIndex
 from algoliasearch_django.decorators import register
+from django.conf import settings
 
 from eahub.profiles.models import Profile
 
+if not settings.IS_ENABLE_ALGOLIA:
```

Comment:
looks like the `not` might be not intended?

Reply:
Doh, got this mixed up, thanks!